reark/src/main/java/io/reark/reark/pojo/IdentifiablePojo.java

```diff
@@ -0,0 +1,123 @@
+package io.reark.reark.pojo;
```

Comment:
Good question, it will have a performance impact. On the other hand, in smallish application such as this one, that should be negligible.

I would say that hand-writing the field comparison is premature optimization as long as it's not a measured issue. If that becomes an issue, it's easy to override the method for those POJOs that specifically need it.